lib/adapters/translate_encoder.js

```diff
@@ -73,13 +72,12 @@ var encoder = processor({
       for (var idx = 0; idx < vals.length; idx += 2) {
         var k = encoder(vals[idx]);
         var v = encoder(vals[idx + 1]);
-        if (k instanceof AMQPSymbol) {
+        result[k] = v;
```

Comment:
yeah but now fields are encoded as `ForcedType('fields', value)`, and `AMQPSymbol` doesn't exist anymore. The encoding information exists from the schema of whatever is being encoded (frame, composite etc), so you'll never get anything but an assignable type here